### PR TITLE
[AC-xx: CI-CD] Create jenkinsfile to deploy to tf cluster 

### PR DESCRIPTION
**Pull request by mable-pmyip**
### Provide screenshots or videos that the function is working.
image

### CODE REVIEW

File: CI-CD/Jenkinsfile.Mable [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            steps {
+                echo 'Cleanup docker space'
+                sh 'docker system prune -f'
+                cleanWs()
+            }
+        }
+
+        stage('Fetch code') {
+            steps{
+                git branch: 'ci-cd/jenkins-tf-Mable ', url: 'https://github.com/A-Comosus/comosus-server.git'
```
Comment (tardigrade425) [inline CI-CD/Jenkinsfile.Mable:23]: fetch这个branch是做什么的？

File: Dockerfile.Mable [hunk 2]
```diff
@@ -0,0 +1,31 @@
+FROM node:alpine AS deps
```
Comment (tardigrade425) [inline Dockerfile.Mable:1]: 这个版本是干什么？

Comment (tardigrade425) [reply Dockerfile.Mable:1]: 固定与其他版本一致

File: Dockerfile.Mable [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# Rebuild the source code only when needed
+FROM node:alpine AS builder
+WORKDIR /app
+COPY . .
+COPY --from=deps /app/node_modules ./node_modules
+RUN yarn prisma generate && yarn build
+
+# Production image, copy all the files and run next
+FROM node:alpine AS runner
```
Comment (tardigrade425) [inline Dockerfile.Mable:14]: 与前端node版本一致

File: CI-CD/Jenkinsfile.Mable [hunk 4]
```diff
@@ -0,0 +1,85 @@
+pipeline {
+    agent any
+
+    tools {
+        nodejs 'node18.4.0'
```
Comment (tardigrade425) [inline CI-CD/Jenkinsfile.Mable]: 确认一下版本

**Review by tardigrade425 — CHANGES_REQUESTED** [review]

File: Dockerfile.Mable (hunk 2 above)
Comment (mable-pmyip) [reply Dockerfile.Mable:1]: I think the other dockerfile user this image ``node:alpine`` as well 
https://github.com/A-Comosus/comosus-server/blob/develop/Dockerfile.Alex#L2
https://github.com/A-Comosus/comosus-client/blob/develop/Dockerfile.Alex#L2

File: Dockerfile.Mable (hunk 3 above)
Comment (mable-pmyip) [reply Dockerfile.Mable:14]: same as this one https://github.com/A-Comosus/comosus-server/blob/develop/Dockerfile.Alex#L18

File: CI-CD/Jenkinsfile.Mable (hunk 1 above)
Comment (mable-pmyip) [reply CI-CD/Jenkinsfile.Mable:23]: for testing only, updated to "develop" branch now

File: CI-CD/Jenkinsfile.Mable (hunk 4 above)
Comment (mable-pmyip) [reply CI-CD/Jenkinsfile.Mable]: turns out this tool isn't needed, I removed it

File: Dockerfile.Mable (hunk 3 above)
Comment (tardigrade425) [reply Dockerfile.Mable:14]: alphine是个简化的版本，要和developer使用的版本保持一致

**Review by tardigrade425 — DISMISSED** [review]

**Review by AlexXin14 — APPROVED** [review]

**Pull request merged**